sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FieldValue.java

```diff
@@ -70,6 +72,9 @@ public FieldValue(final Object value, final FieldValueType valueType) {
             case MAP:
                 formFieldMap = (Map<String, FormField>) value;
                 break;
+            case SELECTION_MARK_STATE:
+                selectionMarkState = (SelectionMarkState) value;
```

Comment:
sorry if I missed it elsewhere, but we said that until the service correctly sends a non-null value in valueSelectionMark, we would return `value.text` in the meantime. Does this code do that?

Reply:
You mean set the value i.e SELECTED / UNSELECTED to value.text?

Reply:
I might have commented in the wrong place, but since for a FormField, valueSelectionMark is returning null, we can in the meantime return the text value for a FormField.value. In python, this would be the string "selected" or "unselected" but maybe this would be an enum in Java

Reply:
Yes. for consistency, i would use the same temp solution that python and .net does.